src/web/lib/contributions.ts

```diff
@@ -68,7 +68,7 @@ export const isRecurringContributor = (isSignedIn: boolean): boolean => {
 // SUPPORT_ONE_OFF_CONTRIBUTION_COOKIE (support cookie, when making one-off contribution)
 // Get the date of the latest one-off contribution by looking at the two relevant cookies
 // and returning a Unix epoch string of the latest date found.
-export const getLastOneOffContributionDate = (): number | undefined => {
+export const getLastOneOffContributionTimestamp = (): number | undefined => {
```

Comment:
Renamed as we want to send a `YYYY-MM-DD` date string to support-dotcom-components which feels like a better fit for the name `getLastOneOffContributionDate`